packages/components/pie-modal/src/index.ts

```diff
@@ -24,7 +29,49 @@ export class PieModal extends RtlMixin(LitElement) {
         headingLevel: ModalProps['headingLevel'] = 'h2';
 
     @query('dialog')
-        _dialog: HTMLDialogElement;
+        _dialog!: HTMLDialogElement;
```

Comment:
Not a change required here but I've changed this in my PR to a `?` which will add a union type to allow undefined to hang off `_dialog`.

Reply:
depending on who merges first I'll take your approach if it works